Marlin/Marlin_main.cpp

```diff
@@ -5877,12 +5996,12 @@ inline void gcode_M400() { stepper.synchronize(); }
   /**
    * M401: Engage Z Servo endstop if available
    */
-  inline void gcode_M401() { deploy_z_probe(); }
+  inline void gcode_M401() { deploy_stow_probe( false ); }
 
   /**
    * M402: Retract Z Servo endstop if enabled
    */
-  inline void gcode_M402() { stow_z_probe(); }
+  inline void gcode_M402() { deploy_stow_probe( true ); }
```

Comment:
Hmmm. Looks as if a macro instead of  `deploy_stow_probe( true/false )` could look nicer. :-)